resources/party.ts

```diff
@@ -15,6 +17,184 @@ const emptyRoleToPartyNames = () => {
   };
 };
 
+// TODO: de fr ko
+const roleLocalized: Record<Role, Partial<Record<Lang, string>>> = {
+  tank: { ja: 'タンク', cn: '坦克' },
+  healer: { ja: 'ヒーラー', cn: '治疗' },
+  dps: { ja: 'DPS', cn: '输出' },
+  crafter: { ja: 'クラフター', cn: '能工巧匠' },
+  gatherer: { ja: 'ギャザラー', cn: '大地使者' },
+  none: { ja: 'すっぴん士', cn: '冒险者' },
+};
+
+// TODO: ja de fr ko
+const jobLocalized: Record<Job, Record<'abbr' | 'full', Partial<Record<Lang, string>>>> = {
```

Comment:
Similar to the above, I know it makes more sense from a code perspective to put these in the same object, but if you separate this into `jobLocalizedAbbr: Record<Job, LocaleText>` and `jobLocalizedFull: Record<Job, LocaleText>` separately then it will show up in the find translations script (and no need for a TODO).